.github/workflows/cli-coexistence_tests.yaml

```diff
@@ -17,24 +17,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v4
 
       - name: Set up Go 1.x
-        uses: actions/setup-go@v3
+        uses: actions/setup-go@v4
```

Comment:
Why this change?

Reply:
Updated to latest versions since few actions were deprecated